include/translate-option.php

```diff
@@ -14,10 +14,24 @@ class PLL_Translate_Option {
 	/**
 	 * Array of option keys to translate.
 	 *
-	 * @var array
+	 * @var string[]
 	 */
 	private $keys;
 
+	/**
+	 * Sanitization callback.
+	 *
+	 * @var string
+	 */
+	private $sanitize_callback;
```

Comment:
```suggestion
	private $sanitize_callback = 'sanitize_option';
```
See later the improvement suggested.